zerver/webhooks/github/view.py

```diff
@@ -54,27 +55,27 @@ def get_opened_or_update_pull_request_body(helper: Helper) -> str:
     payload = helper.payload
     include_title = helper.include_title
     pull_request = payload["pull_request"]
-    action = payload["action"]
+    action = payload["action"].tame(check_string)
     if action == "synchronize":
         action = "updated"
     assignee = None
     if pull_request.get("assignee"):
-        assignee = pull_request["assignee"]["login"]
+        assignee = pull_request["assignee"]["login"].tame(check_string)
     description = None
     changes = payload.get("changes", {})
     if "body" in changes or action == "opened":
-        description = pull_request["body"]
+        description = pull_request["body"].tame(check_string)
```

Comment:
A pull request `body` is [nullable](https://github.com/octokit/webhooks/blob/v5.6.0/payload-schemas/api.github.com/common/pull-request.schema.json#L78) (#22111).